README.md

```diff
@@ -19,9 +19,9 @@ Detailed changelogs are published on the [official osu! site](https://osu.ppy.sh
 ## Requirements
 
 - A desktop platform with the [.NET Core SDK 3.0](https://www.microsoft.com/net/learn/get-started) or higher installed.
-- When running on linux, please have a system-wide ffmpeg installation available to support video decoding.
+- When running on Linux, please have a system-wide ffmpeg installation available to support video decoding.
 - When running on Windows 7 or 8.1, **[additional prerequisites](https://docs.microsoft.com/en-us/dotnet/core/windows-prerequisites?tabs=netcore2x)** may be required to correctly run .NET Core applications if your operating system is not up-to-date with the latest service packs.
-- When working with the codebase, we recommend using an IDE with intellisense and syntax highlighting, such as [Visual Studio 2019+](https://visualstudio.microsoft.com/vs/), [Jetbrains Rider](https://www.jetbrains.com/rider/) or [Visual Studio Code](https://code.visualstudio.com/).
+- When working with the codebase, we recommend using an IDE with IntelliSense and syntax highlighting, such as [Visual Studio 2019+](https://visualstudio.microsoft.com/vs/), [Jetbrains Rider](https://www.jetbrains.com/rider/) or [Visual Studio Code](https://code.visualstudio.com/).
```

Comment:
IntelliSense is actually a [Microsoft trademark](https://trademarks.justia.com/744/22/intellisense-74422731.html). I suggest non-proper phase "intelligent language services". Or replace the whole "intellisense and syntax highlighting", because language services are more about completion and highlighting nowadays.

Reply:
I'd just go with `code completion` if we want to go generic with it.

Reply:
`completion` is about writing code. There are many features about reading and/or refactoring. For new contributors, reading is much more important.

Reply:
I suppose, but `intelligent language services` sounds like corporate newspeak for me.

Reply:
https://en.wikipedia.org/wiki/Intelligent_code_completion

https://en.wikipedia.org/wiki/Syntax_highlighting

I would go with wikipedia terms.